core/src/ast/operator.rs

```diff
@@ -46,31 +47,37 @@ pub enum BinaryOperator {
     BitwiseShiftRight,
 }
 
-impl ToSql for BinaryOperator {
-    fn to_sql(&self) -> String {
+impl Display for BinaryOperator {
```

Comment:
we can use `ToSql`. I understand that it looks just ok to have `Display` impl, but basically we are trying to reduce the use of `From`, `TryFrom` and `Display` like implicit ones.

Reply:
The implementation of `Display` was to implement `Debug`, and having both `ToSql` and `Display` was excessive code duplication.

Reply:
> The implementation of Display was to implement Debug

`Debug` is already implemented for `BinaryOperator`, then we may not need `Display`?

Reply:
Removed.